static/js/components.js

```diff
@@ -0,0 +1,96 @@
+Vue.component(VueQrcode.name, VueQrcode)
+
+Vue.component('satspay-paid', {
+  props: ['charge', 'timer'],
+  template: `
+    <div class="col-sm-10 col-lg-8">
```

Comment:
- the component should not force any col size
- having a `col` at top level also means the parent component has to have a `row`
```suggestion
    <div>
```

Reply:
i did